### ISSUE TITLE
Failed to compile installing v0.2.0 from crates.io

### ISSUE THREAD

**Issue by alpha-tango-kilo** [sloganking/codevis#25]
```
$ cargo install --force --locked codevis

error[E0425]: cannot find value `SIGWINCH` in module `signal_hook::consts`
   --> C:\Users\Atk\.cargo\registry\src\github.com-1ecc6299db9ec823\prodash-20.0.0\src\render\line\engine.rs:234:67
    |
234 |             signal_hook::low_level::register(signal_hook::consts::SIGWINCH, || {
    |                                                                   ^^^^^^^^ not found in `signal_hook::consts`

For more information about this error, try `rustc --explain E0425`.
error: could not compile `prodash` due to previous error
```

Issue appears to be downstream but creating an issue here for your awareness too

Issue in `prodash`: https://github.com/Byron/prodash/issues/12

**Comment by sloganking** [comment]
To confirm, I am going to publish a new version, and you till me if it's fixed for you @alpha-tango-kilo

**Comment by sloganking** [comment]
Try it with `codevis = "0.3.0"`, and if there is a problem I will reopen this issue.

**Comment by alpha-tango-kilo** [comment]
Thanks for the fix, can confirm v0.3.1 successfully installed
